chase/frontend/components/navbar/settingssidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { Sidebar } from "primereact/sidebar";
+import { Card } from "primereact/card";
+import { SelectButton } from "primereact/selectbutton";
+
+interface ColormodeOption {
+  name: string;
+  icon: string;
+  value: string;
+}
+
+export default function SettingsSidebar({
+  settingsSidebarVisible,
+  setSettingsSidebarVisible,
+}) {
+  const [colortheme, setColortheme] = useState<ColormodeOption>({
+    name: "Hell",
+    icon: "pi pi-sun",
+    value: "light",
+  }); // TODO: get this from global state or cookie
+
+  const colortheme_items: ColormodeOption[] = [
+    { name: "Hell", icon: "pi pi-sun", value: "light" },
+    { name: "Dunkel", icon: "pi pi-moon", value: "dark" },
+    { name: "Kontrast", icon: "pi pi-circle-fill", value: "contrast" },
+  ];
+
+  const colorModeTemplate = (option: ColormodeOption) => {
+    return (
+      <>
+        <span className={option.icon} />
+        <span className="p-ml-2 ml-2">{option.name}</span>
+      </>
+    );
+  };
+
+  return (
+    <Sidebar
```

Comment:
The visuals of this need some love but its ok for now :D

Reply:
Haha definitely! 😂
I just wanted to implement it for now so that one has a space to put settings immediately when they become relevant. And tbh aesthetics were not the highest priority in that regard, since it will change drastically anyways 😉